src/ppm.cpp

```diff
@@ -73,6 +73,17 @@
 #include <unistd.h>
 #endif
 
+using namespace std;
+
+const char* getEnvironmentVariable(const char* variableName) {
+    const char* toReturn = getenv(variableName);
+    if (!toReturn) {
+        cout << "Something went wrong attempting to retrieve the environment variable " << variableName << endl;
```

Comment:
Can we swap to the logger?

Reply:
The PPM's logger can be utilized here if we make the getEnvironmentVariable() method a member of the PPM class. I implemented a similar solution for the ACM, so this is definitely possible. I'll go ahead and make those changes.